gmsh_utils/gmsh_IO.py

```diff
@@ -294,9 +301,9 @@ def make_geometry_3d(self, point_coordinates: Union[List[List[float]], npt.NDArr
         Returns:
             None
         """
-
-        surface = self.make_geometry_2d(point_coordinates, name_label, mesh_size)
-        self.create_volume_by_extruding_surface(surface, extrusion_length, name_label)
+        if name_label != "":
```

Comment:
the surface and geometry should always be created, only it should not always be added to a physical group. In other words, remove this if statement